sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -201,7 +204,7 @@ case class Alias(child: Expression, name: String)(
   }
 
   override def sql: String = {
-    val qualifierPrefix = qualifier.map(_ + ".").getOrElse("")
+    val qualifierPrefix = if (qualifier.nonEmpty) qualifier.mkString(".") + "." else ""
```

Comment:
nit: `val qualifierPrefix = qualifier.mkString("", ".", ".")`

Reply:
This won't work for the case when we have Seq.empty.  The suffix "." gets returned even for a empty sequence. 
For a non empty Seq, the above call will be fine. 

Shall we leave the 'if' as is  or is there an equivalent preferred style that would work?

Reply:
ah my bad, I thought it would return empty string for empty seq. Let's leave it.

Reply:
ok, sounds good.